src/components/TitleComponent.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+
+function RenderTitle({title}) {
+    return (
+        <div class="intro">
+            <h2 class="text-center">{title.name}</h2>
+            <h3 class="text-center">{title.description}</h3>
+        </div>
+    )
+}
+
+
+function Title(props) {
+    const title = props.title.map(title => {
```

Comment:
There should just be two html elements here, an h2 and an h3, I don't think we need to map through an array here.

Would be easier to just pass into a title component a few props

<Title title="top section" subTitle="bottom section" />